### PR TITLE
Upgrade to Android SDK 29

### PR DESCRIPTION
**Pull request by mayokunthefirst**
Fixes issue #336

### CODE REVIEW

**Comment by mayokunthefirst** [comment]
Hello @arkivanov I tried checking out the build details for the "Build on macOS" but couldn't see why or where the error is. The drop down refuses to show the details. Also, before I opened this PR, the 3 builds passed on my end here: would appreciate any help concerning this 🙏 

Screenshot 2020-01-02 at 04 21 22

**Comment by arkivanov** [comment]
Restarted the tests, passed

File: sample-android-app/build.gradle [hunk 1]
```diff
@@ -2,13 +2,13 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
```
Comment (CherryPerry) [inline sample-android-app/build.gradle]: Don't jetify appcompat.
Check `Deps` file, change `Deps.appcompat.XXX` artefacts to `Deps.androidx.XXX` and use them in `build.gradle` of sample app.

Comment (mayokunthefirst) [reply sample-android-app/build.gradle]: Alright, will do right away.

Comment (mayokunthefirst) [reply sample-android-app/build.gradle]: @CherryPerry I also changed `Support` to `AndroidX` and `support` to `androidx` in `Deps` file for clarity.

File: gradle.properties [hunk 2]
```diff
@@ -1,9 +1,4 @@
 kotlin.code.style=official
 org.gradle.jvmargs=-Xmx2g -XX:MaxMetaspaceSize=512m -Dfile.encoding=UTF-8
 org.gradle.parallel=true
-org.gradle.caching=true
-# AndroidX package structure to make it clearer which packages are bundled with the
-# Android operating system, and which are packaged with the app's APK
-android.useAndroidX=true
-# Automatically convert third-party libraries to use AndroidX
-android.enableJetifier=true
```
Comment (CherryPerry) [inline gradle.properties]: I think you should still keep it. Picasso uses `android.support` package, so it should be jetified.

https://search.maven.org/artifact/com.squareup.picasso/picasso/2.71828/aar

Comment (mayokunthefirst) [reply gradle.properties]: @CherryPerry  Just to be clear, I should keep both the `useAndroidX` and `enableJetifier` flags ?

Comment (mayokunthefirst) [reply gradle.properties]: Alright.

**Pull request merged**